Makefile

```diff
@@ -323,6 +323,27 @@ image:
 	hack/make-rules/image.sh $(WHAT)
 endif
 
+define CROSS_IMAGE_HELP_INFO
+# Use Buildx multi-architecture docker images.
+#	Usage of Buildx, please ref: https://medium.com/@artur.klauser/building-multi-architecture-docker-images-with-buildx-27d80f7e2408
```

Comment:
We can remove this comment, which has been added in script.

Reply:
fixed.